.github/workflows/doc.yml

```diff
@@ -43,7 +47,7 @@ jobs:
         run: |
           python -m http.server 8880 --directory doc/generated/html &
           sleep 1
-          linkchecker --check-extern http://localhost:8880/svgdigitizer/
+          linkchecker --check-extern http://localhost:8880/svgdigitizer/ --ignore-url="https://repology.org/*"
```

Comment:
Could you add a comment in the workflow why you want to exclude repology?

Reply:
This change was added in #169 since the check passes locally but not in the GitHub workflow.

Reply:
A comment was added.

Reply:
is this the only change you wanted and would it be ready for merging??